client/src/App.js

```diff
@@ -4,21 +4,26 @@ import ProtectedRoute from './components/routes/ProtectedRoute';
 import ChessGame from './components/chess-game/ChessGame';
 import Home from './components/home/Home';
 import Login from './components/login/Login';
-import Dashboard from './components/dashboard/Dashboard';
+// import Dashboard from './components/dashboard/Dashboard';
 import Leaderboard from './components/leaderboard/Leaderboard';
+import MenuBar from './components/menu-bar/MenuBar';
 
 function App() {
   return (
-    <Router>
-      <Switch>
-        <Route path="/login" exact component={Login} />
-        <Route path="/chess-game" exact component={ChessGame} />
-        <Route path="/" exact component={Home} />
-        <ProtectedRoute path="/dashboard" exact component={Dashboard} />
-        <ProtectedRoute path="/leaderboard" exact component={Leaderboard} />
-        <Route path="*" component={() => '404 NOT FOUND'} />
-      </Switch>
-    </Router>
+    <>
+      {/* <MenuBar /> */}
+      <Router>
+        <Route component={MenuBar} />
```

Comment:
Do we need this in `/login` route also?

Reply:
It is menu bar for website, it displays site name and in future can have things like navigation and logo, so I think it should be displayed always.